### PR TITLE
#157814977 Add circleci configuration

### PR DESCRIPTION
**Pull request by code-sleuth**
#### What does this PR do?
This PR adds circleci configuration that will trigger builds.

#### Description of Task to be completed?
Implement continuous integration by way of circleci

#### How should this be manually tested?
This will be run automatically by circleci

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#157814977](https://www.pivotaltracker.com/n/projects/2170023/stories/157814977)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  working_directory: ~/activo-api
+
+cmd_wait_for_postgres: &cmd_wait_for_postgres
+  run:
+    name: Waiting for Postgres to be ready
+    command: |
+      dockerize -wait tcp://localhost:5432 -timeout 1m
+cmd_install_dependencies: &cmd_install_dependencies
+  run:
+    name: Insatll dependencies
```
Comment (kodero) [inline .circleci/config.yml]: Typo??

**Comment by andela-kakpobome** [comment]
Hey @code-sleuth, please ensure you are following the style guides for commit messages, refer to the engineering playbook if you need clarification.

File: .circleci/config.yml (hunk 1 above)
Comment (code-sleuth) [reply .circleci/config.yml]: Fixed

**Comment by andela-kakpobome** [comment]
@code-sleuth there should be a `:` after the `)` and the entire thing should be lowercased

**Comment by code-sleuth** [comment]
Done

**Comment by andela-kakpobome** [comment]
@code-sleuth again, all lowercased

**Comment by code-sleuth** [comment]
All done. Also tested the workflow with @SEUNAGBEYE and everything works as expected.

**Pull request merged**